oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreService.java

```diff
@@ -199,6 +201,16 @@ public class DocumentNodeStoreService {
      */
     private static final String FT_NAME_EMBEDDED_VERIFICATION = "FT_EMBEDDED_VERIFICATION_OAK-10633";
 
+    /**
+     * Feature toggle name to enable full GC mode GAP_ORPHANS cleanup for Mongo Document Store
+     */
+    private static final String FT_NAME_FULL_GC_MODE_GAP_ORPHANS = "FT_FULL_GC_MODE_GAP_ORPHANS_OAK-10896";
+
+    /**
+     * Feature toggle name to enable full GC mode EMPTY_PROPERTIES cleanup for Mongo Document Store
+     */
+    private static final String FT_NAME_FULL_GC_MODE_EMPTY_PROPERTIES = "FT_FULL_GC_MODE_EMPTY_PROPERTIES_OAK-10896";
+
```

Comment:
I was under the impression we decided not to use feature toggles, in which case this (and related code) is not necessary